src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -184,11 +184,11 @@ public static void ServerCertificateValidation_EchoString()
             clientCertThumb = BridgeClientCertificateManager.LocalCertThumbprint; // ClientCert as given by the Bridge
 
             factory = new ChannelFactory<IWcfService>(binding, endpointAddress);
-            factory.Credentials.ServiceCertificate.Authentication.CertificateValidationMode = X509CertificateValidationMode.Custom;
-
-            MyX509CertificateValidator myX509CertificateValidator = new MyX509CertificateValidator(ScenarioTestHelpers.CertificateIssuerName);
-            factory.Credentials.ServiceCertificate.Authentication.CustomCertificateValidator = myX509CertificateValidator;
             factory.Credentials.ServiceCertificate.SslCertificateAuthentication = factory.Credentials.ServiceCertificate.Authentication;
+            //factory.Credentials.ServiceCertificate.SslCertificateAuthentication = new X509ServiceCertificateAuthentication();
```

Comment:
No, the old test is just a workaround of copying the X509ServiceCertificateAuthentication from another property as it can't be constructed yet. Once we have this change available in packages, no workaround will be needed and the old way of doing things isn't something people need to be doing.